### PR TITLE
Add openshift_hosted_registry_storage_swift_insecureskipverify parameter

### PR DESCRIPTION
**Pull request by mcanevet**

### CODE REVIEW

**Comment by mcanevet** [comment]
@vrutkovs done

File: roles/openshift_hosted/templates/registry_config.j2 [hunk 1]
```diff
@@ -56,6 +56,9 @@ storage:
 {%   if openshift_hosted_registry_storage_swift_domainid is defined %}
     domainid: {{ openshift_hosted_registry_storage_swift_domainid }}
 {%   endif -%}
+{%   if openshift_hosted_registry_storage_swift_insecureskipverify is defined %}
```
Comment (vrutkovs) [inline roles/openshift_hosted/templates/registry_config.j2]:
```suggestion
{%   if openshift_hosted_registry_storage_swift_insecureskipverify | default(false) | bool %}
```

**Pull request merged**